.github/workflows/build.yml

```diff
@@ -0,0 +1,178 @@
+name: build
+
+on:
+  push:
+    branches:
+      - 'master'
+      - 'sq-*'
```

Comment:
Any reason for only building on push to these these branches? Whilst the PR job will pick up the build if targeting master or one of the (unreleased) support branches, I'd actually expect to run LTS releases from `release/*` which wouldn't be covered under these patterns. Ideally I'd run the action on all PRs and Pushes, but didn't know if there was a reason you'd applied these restrictions.

Reply:
@mc1arke Yes you're right we could build for all branches that's not a problem.